server/fleet/scripts.go

```diff
@@ -282,6 +282,22 @@ func (hsr HostScriptResult) AuthzType() string {
 	return "host_script_result"
 }
 
+type BatchScriptHost struct {
+	// ID is the host on which the script was executed.
+	ID uint `json:"id" db:"id"`
+	// Display name is the host's display name.
+	DisplayName string `json:"display_name" db:"display_name"`
+	// ExecutionID is a unique identifier for a single execution of the script.
+	ScriptExecutionID string `json:"script_execution_id" db:"execution_id"`
+	// Output is the combined stdout/stderr output of the script. It is empty
+	// if no result was received yet.
+	ScriptOutput string `json:"script_output_preview,omitempty" db:"output"`
+	// Executed at is the time the script was executed on the host (if at all).
+	ScriptExecutedAt *time.Time `json:"script_executed_at,omitempty" db:"updated_at"`
```

Comment:
This is not referenced anywhere in code (so no new `nil` checks needed), but handling it as a string like I was made it not come out as an ISO timestamp which caused confusion on the frontend.